src/movie-components/movie-container-component/movie-container.js

```diff
@@ -2,15 +2,13 @@ import React from 'react';
 import MovieCard from '../movie-card-component/movie-card-component.js';
 
 function MovieContainer (props)  {
-  let keys = Object.keys(props)
-  let movies = keys.map(key => props[key])
   return (
-    <>
-    {movies.map(movie => {
-      return < MovieCard {...movie} key={movie.id} />
+    <article className="all-movies-container">
+    {props.movies.map(movie => {
```

Comment:
I'd maybe pull this map out into its own variable (this is called memoizing), and then reference that variable in your return. Keep the return and renders very clean and readable!